routes/adminapi.js

```diff
@@ -61,23 +61,27 @@ router.post('/saveword', function(req, res) {
     if (!(config.COOKIE_NAME in req.cookies))
       return res.status(400).send('Cookie not provided.');
     var sessionId = req.cookies[config.COOKIE_NAME];
+    db.getUser(sessionId, (user) => {
+      if (user === false || user === 'No User.') return res.status(500).send('Error Occurred');
```

Comment:
yeah so basically this works, but i guess you must have realised by now how ugly this if/else checks looks.
may be we can come back to this later and convert this to `Promises`